addon/helpers/flatten.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+const {
+  Helper: { helper }
+} = Ember;
+
+export function flatten(array) {
+  array = array || [];
```

Comment:
Can we have this as `array || [array];`? I would prefer that this method always returns an array regardless of the value passed into it.

Also, we should be using an `isArray` check here. It is still valid if the array contains `null` or `undefined` values.